### ISSUE TITLE
Which atmospheric correction should be used with MARIDA dataset? 

### ISSUE THREAD

**Issue by giustiandrea91** [marine-debris/marine-debris.github.io#4]
Hi,
I am using MARIDA to perform a random forest classification. I have a doubt about the atmospheric correction to apply in ACOLITE to the S2 tiles before performing the classification. As stated in Kikaki et al. (2022) rayleigh reflectance values are extracted, while in Kikaki (2020) surface reflectance values are obtained. 
Could you please confirm exactly which ACOLITE's outputs should I obtain to obtain the best classification results? rhos_*: ρs, the surface level reﬂectance or rhorc_*: ρrc, the Rayleigh corrected reﬂectance ? 

Thank you for you support.
Andrea

**Comment by KateKikaki** [comment]
Hi Andrea,

Thank you for your message. Indeed, in our first work (Kikaki et al., 2020) we extracted the surface reflectance values, to demonstrate the different spectral signatures of marine debris and Sargassum.  In MARIDA, following the methods of past studies (Qi et al., 2020 https://doi.org/10.1016/j.rse.2020.111659) related to floating materials detection, we used the Rayleigh reflectance values. I would suggest using Rayleigh reflectance values to avoid potential errors caused by atm. correction (aerosol corrections). Note that MARIDA data are already corrected (Rayleigh).

Let me know if you have any questions,
Katerina
